tests/functional/adapter/test_source_freshness.py

```diff
@@ -0,0 +1,52 @@
+import os
+import pytest
+
+from dbt.tests.util import get_artifact, run_dbt
+
+freshness_via_metadata_schema_yml = """version: 2
```

Comment:
Nit, can we move `version: 2` to the following line?
```suggestion
freshness_via_metadata_schema_yml = """
version: 2
```